docs/getting-started.md

```diff
@@ -6,7 +6,8 @@ development cluster.
 ## Building
 
 The KubeVirt build system runs completely inside docker. In order to build
-KubeVirt you need to have `docker` and `rsync` installed.
+KubeVirt you need to have `docker` and `rsync` installed. You also need to have `docker`
+running, and have the permissions to access it.
```

Comment:
Can you also please add the link to https://docs.docker.com/install/linux/linux-postinstall/